Source/ProjectFirma.Web/Views/Program/ProjectBlockListGridSpec.cs

```diff
@@ -41,7 +40,11 @@ public ProjectBlockListGridSpec(Person currentPerson, Models.Program currentProg
 
             Add(Models.FieldDefinition.ProjectIdentifier.ToGridHeaderString(),
                 x => (x.Project != null) ? x.Project.ProjectGisIdentifier : x.ProjectGisIdentifier,
-                300, DhtmlxGridColumnFilterType.Html);
+                200, DhtmlxGridColumnFilterType.Html);

```

Comment:
FilterType.Html should probably be .Text. Html will treat the cells as HtmlString. and I think ProjectGidIdentifier is just a string

Reply:
Fixed.